insights/parsers/partitions.py

```diff
@@ -0,0 +1,77 @@
+"""
+Partitions - file ``/proc/partitions``
+======================================
+
+This parser reads the ``/proc/partitions`` file, which contains
+partition block allocation information.
+
+"""
+from collections import namedtuple
+
+from insights import Parser, parser, LegacyItemAccess
+from insights.parsers import SkipException, ParseException
+from insights.specs import Specs
+
+
+@parser(Specs.partitions)
+class Partitions(Parser, LegacyItemAccess):
```

Comment:
The `LegacyItemAccess` only works when `self.data` is a dictionary, but here it's a list of dicts.

Reply:
Done